src/forms/stock-dispense/stock.resource.tsx

```diff
@@ -0,0 +1,67 @@
+import useSWR from 'swr';
+import { openmrsFetch, useSession } from '@openmrs/esm-framework';
+import { type StockDispenseRequest, type InventoryItem, type MedicationDispense } from '../../types';
+import { getUuidFromReference } from '../../utils';
+
+//TODO: Add configuration to retrieve the stock dispense endpoint
+// For stock dispense to work, stock management module should be installed and configured
+/**
+ * Fetches the inventory items for a given drug UUID.
+ *
+ * @param {string} drugUuid - The UUID of the drug.
+ * @returns {Array} - The inventory items.
+ */
+export const useDispenseStock = (drugUuid: string) => {
+  const session = useSession();
+  const url = `/ws/rest/v1/stockmanagement/stockiteminventory?v=default&totalCount=true&drugUuid=${drugUuid}&includeBatchNo=true&groupBy=LocationStockItemBatchNo&dispenseLocationUuid=${session?.sessionLocation?.uuid}&includeStrength=1&includeConceptRefIds=1&emptyBatch=1&emptyBatchLocationUuid=${session?.sessionLocation?.uuid}&dispenseAtLocation=1`;
```

Comment:
Not to weigh into this after the fact, but, thinking about the TAC call we are on right now, it would be great if, long-term, the stock management module support an FHIR endpoint so that it would hopefully be more standardized.